profiles/arch/amd64/package.use.mask

```diff
@@ -17,6 +17,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Andrew Ammerlaan <andrewammerlaan@gentoo.org> (2023-12-18)
+# Experimental and only available on amd64, x86, arm64
```

Comment:
Let's not list arches because we won't be updating all of them afterwards. Just say "subset of architectures" or so.